independent-publisher-connectors/JiraSearch/apiDefinition.swagger.json

```diff
@@ -0,0 +1,392 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "JIRA Search",
+    "description": "Searches for JIRA issues using JQL expression using the GET pattern. More information can be found at: https://docs.atlassian.com/software/jira/docs/api/REST/8.16.2/#api/2/search-search",

```

Comment:
@paulculmsee The description value can be improved by adding some more information about to your connector product or service.

Documentation - [Description](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)

Reply:
Added more detail